ajax/libs/socket.io/package.json

```diff
@@ -12,10 +12,9 @@
   "npmName": "socket.io-client",
```

Comment:
@x09326 but the npm auto-update config added in 4cce6db0d22426a0c0875a84d264658cb348116a. It use `socket.io-client`.